File: jira-gpt-tests-java/src/main/java/qa/hack/jira/JiraTestExtractor.java
package qa.hack.jira;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class JiraTestExtractor {

	Logger log = LoggerFactory.getLogger(JiraTestExtractor.class);

	public String JIRA_URL;
	public String JIRA_TOKEN;

	public JiraTestExtractor(String JIRA_URL, String JIRA_TOKEN) {
		log.debug("creating instance");
		this.JIRA_URL = JIRA_URL;
		this.JIRA_TOKEN = JIRA_TOKEN;
	}


}

File: jira-gpt-tests-python/jira-test-extractor.py
import os

from jira import JIRA 
from dotenv import load_dotenv

load_dotenv()

JIRA_URL = os.getenv("JIRA_URL")
JIRA_USERNAME = os.getenv("JIRA_USERNAME")
JIRA_TOKEN = os.getenv("JIRA_TOKEN")

print("JIRA_URL:", JIRA_URL)
print("JIRA_USERNAME:", JIRA_USERNAME)
print("JIRA_TOKEN:", JIRA_TOKEN)